oap-server/server-query-plugin/mqe-grammar/src/main/antlr4/org/apache/skywalking/mqe/rt/grammar/MQELexer.g4

```diff
@@ -72,6 +73,7 @@ DES options { caseInsensitive=true; }: 'des';
 
 // Literals
 INTEGER: Digit+;
+STRING_PARAM: '\'' NameLetter+ '\'';
```

Comment:
if the params are `sum/max/avg..`, should be better to define them rather than use `String`. Can refer to the `order` https://github.com/apache/skywalking/blob/de3a49ef7b5631f4930d33c0f33b3ce07f461af9/oap-server/server-query-plugin/mqe-grammar/src/main/antlr4/org/apache/skywalking/mqe/rt/grammar/MQEParser.g4#L70

Reply:
Ok，thanks. I will fix these.